lib/src/ui/pluto_left_frozen_rows.dart

```diff
@@ -71,3 +71,108 @@ class PlutoLeftFrozenRowsState
     );
   }
 }
+
+class FrozenRow extends StatefulWidget {
```

Comment:
Do you really need to create a new class for this?
Isn't it possible to just separate into methods in the class that already exists?